dapla-manual/statistikkere/jupyter-pyspark.qmd

```diff
@@ -21,60 +21,7 @@ Før man starter **Jupyter-pyspark** bør man ha lest [kapitlet om Dapla Lab](./
-
-@fig-dapla-lab-konf-buckets-data-admins viser en bruker som aktiverer sin *data-admins* tilgang for team *dapla-felles*. Hvis brukeren ikke oppgir en begrunnelse vil de få en feilmelding ved oppstart av tjenesten.  
-
-![Aktivere tilgang til kildedata for data-admins.](../images/dapla-lab-konf-buckets-data-admin.png){fig-alt="Viser tjenestekonfigurasjonen i Dapla Lab." #fig-dapla-lab-konf-buckets-data-admins}
-
-### Ressurser
-
-Under fanen **Ressurser** kan man velge hvor mye CPU og RAM man ønsker i tjenesten, slik som vist i @fig-dapla-lab-resources. Velg så lite som trengs for å gjøre jobben du skal gjøre. Spark tildeles nå automatisk 90% av minnet konfigurert i tjenestekonfigurasjonen.
-
-![Konfigurasjon av ressurser for Jupyter-pyspark i Dapla Lab](../images/dapla-lab-resources.png){fig-alt="Viser Resources-fanen i Jupyter-pyspark-konfigurasjonen i Dapla Lab." #fig-dapla-lab-resources}
-
-
-### Diskplass
-
-Som default får alle som starter en instans av Jupyter-pyspark en lokal disk på **10GB** inne i tjenesten. Under *Diskplass*-fanen kan man velge å øke størrelsen på disken eller ikke ha noe disk i det hele tatt. Siden lokal disk i tjenesten hovedsakelig skal benyttes til å lagre en lokal kopi av koden som lagres på GitHub mens man gjør endringer, bør ikke størrelsen på disken være større enn nødvendig. @fig-dapla-lab-persistence viser valgene som kan gjøres under Diskplass-fanen. 
-
-![Konfigurasjon av lokal disk for Jupyter-pyspark i Dapla Lab](../images/dapla-lab-persistence.png){fig-alt="Viser Persistence-fanen i Jupyter-pyspark-konfigurasjonen i Dapla Lab." #fig-dapla-lab-persistence}
-
-### Git
-
-Under fanen **Git** kan man konfigurere Git og GitHub slik at det blir lettere å jobbe med inne i tjenesten. Som standard arves informasjonen som er lagret under [*Min konto*-*Git* i Dapla Lab](./dapla-lab.html#git). Informasjonen under tjenestekonfigurasjonen blir tilgjengeliggjort som miljøvariabler i tjenesten. Informasjonen blir også lagt i `$HOME/.netrc` slik at man kan benytte det uten å gjøre noe mer for å jobbe mot GitHub fra tjenesten.  
-
-![Konfigurasjon av Git for Jupyter-pyspark i Dapla Lab](../images/dapla-lab-konf-git.png){fig-alt="Viser Git-fanen i Jupyter-pyspark-konfigurasjonen i Dapla Lab." #fig-dapla-lab-konf-git}
-
-@fig-dapla-lab-konf-git viser at brukeren som standard får aktivert **Aktiver Git**. Dette innebærer at Git-brukernavn, Git e-post og GitHub-token arves fra brukerkonfigurasjonen. I tillegg opprettes SSBs standard Git-konfigurasjon i `~/.gitconfig`. 
-
-## Konfigurering av PySpark-ressurser i Notebook
-
-Fra og med den siste oppdateringen trenger du ikke lenger å manuelt angi mengden RAM i PySpark-konfigurasjonen. Tjenesten er nå konfigurert slik at Spark automatisk tildeles 90% av det totale minnet som er definert i tjenestekonfigurasjonen under **Ressurser**.
-
-Dette forenkler oppsettet og sikrer optimal utnyttelse av tilgjengelige ressurser.
-
-
```

Comment:
I think the jupyter-pyspark specific configuration should still be included. CC: @ssb-jnk